plug.py

```diff
@@ -401,6 +401,32 @@ def notify_capidata(
     return error
 
 
+def notify_capi_fleetcarrierdata(
```

Comment:
I don't think `is_beta` is actually necessary in this case.  The `.source_host` should denote it (because if there's ever a beta of the game it'll be `pts-companion.orerve.net`), and if somehow the beta detection has failed and the live host was used anyway, then the data will be for the live game, not beta.

Reply:
Yeah, was following the pattern of the other plugin callbacks, but agree everything is in the `source_host`. Will change.